### PR TITLE
Created speedblog for our project

### PR DESCRIPTION
**Pull request by afzal442**
For [#36](https://github.com/open-life-science/ols-3/issues/36)

### CODE REVIEW

File: _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# OLS-3 Speed Blog: Documentation enhancement with open science practices at sktime (open-source project), Afzal Ansari, Abdulelah Al Mesfer
```
Comment (bebatut) [inline _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md:1]:
```suggestion
---
layout: post
title: "OLS-3 Speed Blog: Documentation enhancement with open science practices at sktime (open-source project)"
authors:
- afzal442
- abdulelahsm
image: https://i.imgur.com/Ij3js3V.png
---
 
*This post is guest authored by Afzal Ansari and Abdulelah Al Mesfer, two recent graduates of OLS-3.*
```

File: _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+
+## Project background ![](https://i.imgur.com/Ij3js3V.png)
+
+sktime is a new Python toolbox for machine learning with time series (https://github.com/alan-turing-institute/sktime). It provides state-of-the-art time series algorithms and scikit-learn compatible tools for building, tuning and evaluating complex models.
```
Comment (bebatut) [inline _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md:7]:
```suggestion
[sktime](https://github.com/alan-turing-institute/sktime) is a new Python toolbox for machine learning with time series. It provides state-of-the-art time series algorithms and scikit-learn compatible tools for building, tuning and evaluating complex models
```

File: _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+The goal was to make it easier for users and other developers to directly get in touch with the algorithm experts **to ask questions or suggest code improvements** and to recognize their contributions more visibly and formally **to encourage long-term maintenance of their contributions**.
+
+## Key understanding and accomplishments
+
+We gained useful insights into this project development through our journey. I listed few of them like we learned Open Community of Practice (Openness, Collaborations, Agile open software development tools, licensing, open leadership, etc.) and web development stacks. 
+
+## The main goals achieved in this project
+
+We acheived how to use open Science practices in developing resources and apply OLS principles to open leadership and working open in their projects and communities.
+Secondly, we were able to create dynamic search table [gh issue link](https://github.com/alan-turing-institute/sktime/issues/704) for users and other developers at sktime to find the algorithm experts as an estimator overview.
```
Comment (bebatut) [inline _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md:28]:
```suggestion
Secondly, we were able to create dynamic search table [GitHub issue link](https://github.com/alan-turing-institute/sktime/issues/704) for users and other developers at sktime to find the algorithm experts as an estimator overview.
```

File: _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## What elements helped you get there?
+
+During our cohort calls, experts used to discuss various topics under OLS principles which helped in building our foundation of our project.
+And also we had our mentor-mentee call scheduled for an half-an-hour every two weeks where we used to discuss our project progress and accomplish our milestones set for our project. 
+
+
+## Next steps
+With the help of our Mentor Toby Hogdes, we have created our dynamic table. We are also working on a few other things as mentioned below:
+* Add other informations like literature reference and links class name of each module to their API reference
+* Use github action for updating CODE-OWNER’s file
```
Comment (bebatut) [inline _posts/2021-06-28-ols-3-participant-Afzal-Ansari-speedblog.md:39]:
```suggestion
* Use GitHub action for updating CODE-OWNER's file
```

**Review by bebatut — COMMENTED** [review]
Thanks @afzal442. It looks good. I just made some suggestions (the most important one is about the metadata on the top that are required)

**Comment by afzal442** [comment]
Thanks a lot, @bebatut for the review!  Will resolve as per your suggestions.

**Comment by bebatut** [comment]
Thanks @afzal442. It is online: https://openlifesci.org/posts/2021/06/28/ols-3-participant-Afzal-Ansari-speedblog/

**Comment by afzal442** [comment]
That's great. Thanks again for your kind support especially.

**Pull request merged**